pkg/cli/kubernetes/logstream/types.go

```diff
@@ -32,6 +34,9 @@ type Options struct {
 	// KubeContext is the name of the kubernetes context to use for connection.
 	KubeContext string
 
+	// KubeClient is the kubernetes client to use for connection.
```

Comment:
If `KubeContext` is unused can we remove it?

Reply:
Removed the **KubeContext**.